opensearch_py_ml/sentence_transformer_model/sentencetransformermodel.py

```diff
@@ -0,0 +1,722 @@
+# SPDX-License-Identifier: Apache-2.0
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+# Any modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+
+import os
+import pickle
+import random
+import shutil
+import subprocess
+import time
+from typing import List
+from zipfile import ZipFile
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import torch
+import yaml
+from accelerate import Accelerator, notebook_launcher
+from sentence_transformers import InputExample, SentenceTransformer
+from torch.utils.data import DataLoader
+from tqdm import tqdm
+from transformers import TrainingArguments, get_linear_schedule_with_warmup
+
+
+class SentenceTransformerModel:
+    def __init__(self, model_id: str = None) -> None:
+        """
+        Description:
+        Initiate a sentence transformer model object.
+
+        Parameters:
+        model_id: str = None
+             the url to download sentence transformer model, if None, default as 'sentence-transformers/msmarco-distilbert-base-tas-b'
```

Comment:
```suggestion
             the huggingface mode id to download sentence transformer model, if None, default as 'sentence-transformers/msmarco-distilbert-base-tas-b'
```

Reply:
modify docstring in @mingshl
[edit docstring](https://github.com/opensearch-project/opensearch-py-ml/pull/18/commits/0b8f8f17968f637c879114906b598c9eb0dd7252)